packages/privacy-toolset/src/cookie-banner/granular-consent.tsx

```diff
@@ -23,10 +19,15 @@ export const GranularConsent = ( {
 	);
 	return (
 		<div className="cookie-banner__bucket-container">
-			<FormToggle onChange={ handleChange } checked={ checked } disabled={ disabled } />
+			<FormToggle
+				onChange={ handleChange }
+				checked={ checked }
+				disabled={ disabled }
+				data-testid={ `${ name }-bucket-toggle` }
```

Comment:
In general, I agree that we should restrain from using `data-testid` as it's a "shortcut". However, in this case, the input is pretty custom (i.e. hard to identify because of its HTML structure) so this kind of testid helped me to focus on the core of the test, rather than writing the long selector (I would need to find an input that is a sibling of a text).

It's not my go-to approach but I believe this is one of the exceptions we can actually depend on `data-testid`.